Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -756,6 +756,7 @@ public extension _ChatChannelController {
     /// - Parameters:
     ///   - text: Text of the message.
     ///   - pinning: Pins the new message. `nil` if should not be pinned.
+    ///   - isSilent: A flag indicating whether the message is a silent message.
```

Comment:
nit: Can you add the following to the doc:
```
Silent messages are special messages that don't increase the unread messages count nor mark a channel as unread. 
```

Reply:
Added it 👍